sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_component_builder.py

```diff
@@ -271,22 +272,24 @@ def _map_type(_meta):
 
             # Note: Here we set PipelineOutput as Pipeline's output definition as we need output binding.
             output_meta = Output(
-                type=_map_type(meta), description=meta.description, mode=meta.mode, is_control=meta.is_control
+                type=_map_type(meta),
+                description=meta.description,
+                mode=meta.mode,
+                is_control=meta.is_control
             )
             pipeline_output = PipelineOutput(
                 port_name=key,
                 data=None,
+                # This is a hack: when parallel_for node outputs promoted to pipeline level,
+                # pipeline outputs type should be same with node levle.
```

Comment:
typo here: levle => level

Reply:
and is this comment for change in this PR? seems like comment for previous change?

Reply:
Good catch!

Reply:
It's newly added, I can't remember why we need to add meta here.

Reply:
Updated